db/migrate/20210121170546_add_creator_id_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatorIdToEvents < ActiveRecord::Migration[6.1]
+  def change
+    add_column :events, :creator_id, :integer
```

Comment:
Kindly add the index to the creator_id. The index is used by the rails app to optimize for fast lookup.